.ci/bump-version.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+if [ $# -lt 1 ]; then
+  echo "usage: ${0} folder"
+  exit 1
+fi
+AGENT_VERSION="${1}"
+
+# Install Go using the same travis approach
```

Comment:
probably we can put this script in the library https://github.com/elastic/beats/blob/b14f798d000f66582d37487bbda1fe9de6999e7d/.ci/scripts/install-go.sh in most of the places we use [gvm](https://github.com/andrewkroh/gvm)

Reply:
The reason for using the travis approach was to align the same commands in travis and Jenkins for the apm-agent-go.

I did copy the logic from the apm-agent-go, so no strong opinion to change it and use something else as there is no requirements to use any specific approach in this repo.

Reply:
The Go agent uses some features of gimme that are not available in gvm as far as I'm aware:
 - get the current `stable` release
 - get the most recent patch release of a specified minor (e.g. `1.12.x`)
 - build the current `master` from source